docs/install.rst

```diff
@@ -88,6 +68,56 @@ GitHub
       $ heroku config:set GITHUB_BLENDED_PROJECT=edx:9
 
 
+Jira
+~~~~
+
+The bot can be configured to work with many Jira servers.
+
+The JIRA_INFO_FILE setting specifies a YAML file name in openedx-webhooks-data repo.
+For a hypothetical organization named MegaCorp, it might look like this:
+
+    .. code-block:: yaml
+
+        # The key is a short name that will be looked up from the label.
+        # This configures the settings for a "jira:mega" label.
+        Mega:
+            # The URL of the Jira instance.
+            server: https://megacorp.atlassian.net
+            # The email address and user token for the account that will make
+            # Jira issues.  Be sure it has the privileges needed.
+            email: bot-account@megacorp.com
+            token: 84Ma................z1FC
+            # The URL of a YAML file mapping repos to Jira project details.
+            mapping: https://raw.githubusercontent.com/megacorp/.github/HEAD/jira-mapping-prod.yaml
+            # A textual description of the Jira instance, to use in comments.
+            # It will be used in a sentence like this:
+            # "I've created issue PROJ-123 in {description}."
+            description: the private MegaCorp Jira
+
+The mapping file is YAML that maps a repo name to two pieces of information: a
+Jira project and an issue type to create in that project. The file can be at
+any URL.  As shown above, a raw GitHub URL is convenient.  The file specifies
+repos, possibly with wildcards, and for each provides the project and issue
+type to use:
```

Comment:
```suggestion
The mapping file is YAML that maps a repo name to two pieces of information: a
Jira project and an issue type to create in that project. The file can be at
any URL.  As shown above, a raw GitHub URL is convenient.  The reason this is 
separate from other config is so that this mapping file can be managed by Megacorp
while the info file above with credentials in it is managed by the team running 
openedx-webhooks. The file specifies repos, possibly with wildcards, and for each
provides the project and issue type to use:
```

It might be useful to add a note about why there are two files here.